pkg/certificate/manager.go

```diff
@@ -117,9 +118,16 @@ func (m *Manager) start(ctx context.Context, mrcClient MRCClient) error {
 func (m *Manager) handleMRCEvent(mrcClient MRCClient, event MRCEvent) error {
 	switch event.Type {
 	case MRCEventAdded:
-		mrc := event.MRC
+		mrc := event.NewMRC
+
+		// ignore add event if MRC has no state
+		if mrc.Status.State == "" {
+			log.Debug().Msgf("received MRC add event for MRC %s. Ignoring because MRC has no state", mrc.GetName())
+			return nil
+		}
+		// ignore add event if MRC state is error
 		if mrc.Status.State == constants.MRCStateError {
-			log.Debug().Msgf("skipping MRC with error state %s", mrc.GetName())
+			log.Debug().Msgf("received MRC add event for MRC %s. Skipping MRC with error state", mrc.GetName())
```

Comment:
would we want to raise an event here?

Reply:
I am still working through how to respond an MRC with an error state